nifi-extension-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-client-service/src/main/java/org/apache/nifi/elasticsearch/ElasticSearchClientServiceImpl.java

```diff
@@ -336,7 +335,9 @@ private void verifySniffer(final ConfigurationContext context, final RestClient
     }
 
     private List<String> getElasticsearchRoot(final RestClient verifyClient, final OAuth2AccessTokenProvider tokenProvider) throws IOException {
-        final Request request = addJWTAuthorizationHeader(new Request("GET", "/"), tokenProvider);
+        final Request request = new Request("GET", "/");
+        addJWTAuthorizationHeader(request.getOptions().toBuilder(), tokenProvider);
```

Comment:
Need to `setOptions` on the `request` once the request option builder has been updated by `addJWTAuthorizationHeader`

Reply:
Addressed mistake made after `rebase` from `main`